sessions/sessions.go

```diff
@@ -2194,20 +2239,25 @@ func (sS *SessionS) BiRPCv1InitiateSessionWithDigest(clnt rpcclient.RpcClientCon
 
 // NewV1UpdateSessionArgs is a constructor for update session arguments
 func NewV1UpdateSessionArgs(attrs, acnts bool,
```

Comment:
Please group here the arguments to look a bit more organized (ie: attributes, attributesIDs, ....)